Frontend/ui-customer/src/Screens/Handlers.purs

```diff
@@ -63,5 +63,8 @@ import Screens.TicketBookingFlow.MetroTicketBooking.Handler (metroTicketBookingS
 import Screens.TicketBookingFlow.MetroTicketStatus.Handler (metroTicketStatusScreen) as UI
 import Screens.RentalBookingFlow.RentalScreen.View (rentalScreen) as UI
 import Screens.RideBookingFlow.PickupInstructionsScreen.Handler (pickupInstructionsScreen) as UI
+import Screens.RiderDriverProfileScreen.Handler (driverProfileScreen) as UI
 import Screens.NammaSafetyFlow.Handler (nammaSafetyScreen) as UI
 import Screens.DataExplainWithFetch.Handler (dataFetchScreen) as UI
+import Screens.RideBookingFlow.RiderRideCompletedCard.Handler (riderRideCompletedScreen) as UI
```

Comment:
Import not used

Reply:
no we are using this in flow.purs to call this screen